src/components/dashboard/Who.js

```diff
@@ -49,6 +51,28 @@ const Who = (props: AmountProps) => {
     [screenProps]
   )
 
+  const canContinue = useCallback(async () => {
+    // =====
+    // await for android keyboard to be closed before redirecting to the next step
+    if (isAndroidNative) {
+      await new Promise(res => {
```

Comment:
we should remove listener after keyboard hidden otherwise we've got memory leaks:
```javascript
await new Promise(resolve => {
  const keyboardDidHide = () => {
    Keyboard.removeListener('keyboardDidHide', keyboardDidHide)
    resolve()
  }

  Keyboard.addListener('keyboardDidHide', keyboardDidHide)
  Keyboard.dismiss()
}) 
```